src/app/components/hit/hit.component.spec.ts

```diff
@@ -0,0 +1,186 @@
+import { TestBed } from '@angular/core/testing';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { HitComponent } from './hit.component';
+import { ImgixAspectRatioPipe } from 'src/app/pipes/imgix-aspect-ratio.pipe';
+
+class MockImgix {
+  init(obj) {
+    return obj;
+  }
+}
+
+describe('HitComponent', () => {
+  let fixture;
+  let component;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        HitComponent,
+        ImgixAspectRatioPipe
+      ],
+      providers: [],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
+    }).compileComponents();
+    fixture = TestBed.createComponent(HitComponent);
+    component = fixture.debugElement.componentInstance;
+    component.hit = {};
+
+    window['imgix'] = new MockImgix(); // imgix is globally defined by another script
+  });
+
+  it('should create a component', async () => {
+    expect(component).toBeTruthy();
+  });
+
+  describe('Tests hit.image states', () => {
+    it('should have 3 blocks if hit.image is truthy: image link, text, meta', () => {
```

Comment:
```suggestion
    it('should have image link, text, meta if hit.image is truthy', () => {
```
@s-henney 
This is more succinct and the count of blocks is unnecessary.

What might be better is if you could preserve state and run three tests for hit.image is truthy that individually checks if image link, text, meta are available if truthy. Not sure if that's possible or appropriate for this CR.

Reply:
if you agree with the above suggestion, you could apply it to the other tests in this file